File: 1. Begginer/11- Pida al usuario dos números enteros (que se guardarán en las variables num1 y num2).Si num2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resto de l/main.php
<?php
$num1 = (int)readline("Ingrese el primer número entero: ");
$num2 = (int)readline("Ingrese el segundo número entero: ");
if ($num2 == 0) {
	echo "Error: NO se puede dividir un número entre 0";
} else {
	echo "El resultado de dividir $num1 entre $num2 es igual a: " . (float) $num1 / $num2;
}

File: 1. Begginer/11- Pida al usuario dos números enteros (que se guardarán en las variables num1 y num2).Si num2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resto de l/main.py
num1 = int(input("Ingrese el primer número entero: "))
num2 = int(input("Ingrese el segundo número entero: "))
if (num2 == 0):
	print("Error: no se puede dividir un número entre 0")
else:
	print("El resultado de dividir " + str(num1) + " entre " + str(num2) + " es igual a: " + str(num1/num2))